rdflib/compat.py

```diff
@@ -6,29 +6,34 @@
 import re
 import codecs
 import warnings
+import typing as t
 
-
-# clean ElementTree import
-try:
-    from lxml import etree
-except ImportError:
+if t.TYPE_CHECKING:
+    import xml.etree.ElementTree as etree
+else:
     try:
-        # Python 2.5
-        import xml.etree.cElementTree as etree
+        from lxml import etree
     except ImportError:
         try:
             # Python 2.5
-            import xml.etree.ElementTree as etree
+            import xml.etree.cElementTree as etree
         except ImportError:
             try:
-                # normal cElementTree install
-                import cElementTree as etree
+                # Python 2.5
```

Comment:
Can this be removed? It looks like the support for Python 2.5 is retained here

Reply:
I think it can be simplified, I will update the PR to remove it tonight

Reply:
Simplified to this:

```python
    try:
        from lxml import etree
    except ImportError:
        import xml.etree.ElementTree as etree
```